compiler/Cargo.lock

```diff
@@ -52,13 +52,13 @@ checksum = "9b34d609dfbaf33d6889b2b7106d3ca345eacad44200913df5ba02bfd31d2ba9"
 
 [[package]]
 name = "async-trait"
-version = "0.1.58"
+version = "0.1.71"
```

Comment:
Could we omit this `Cargo.lock` change from the PR?

Reply:
Sure, I just included it, since the pipeline would fail otherwise.